.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules/
 contracts/
 .nyc_output
 coverage/
+.vscode/
+src/.watch.yml
+coverage.lcov
```

Comment:
They are necessary to ignore.
`.vscode/` because of the visual studio debugging mode
`src/.watch.yml` no need to keep update the changes any more.
`coverage.lcov` no need